documentation/flows/agent/ap_metrics.md

```diff
@@ -62,16 +95,39 @@ AP Metrics Query should pass the following verifications:
 * Verify that the controller sends a correctly formatted AP Metrics Query message.
 * Verify that the message includes one or more BSSIDs operated by agent.
 
-AP Metrics response should pass the following verifications:
+AP Metrics Response should pass the following verifications:
 
 * Verify that a correctly formatted AP Metrics Response message is sent by the agent.
 * Verify that the MID of the agent AP Metrics Response message matches the MID of the controller AP Metrics Query message.
 * Verify that the agent transmits the AP Metrics Response message within 1 second of receipt of the AP Metrics Query message.
 * Verify that the agent includes one AP Metrics TLV in the response for each of the requested BSSIDs in the query message.
 * Verify that each of the AP Metrics TLV is decodable and includes all fields per TLV definition in [1]
 
-For the second response (after the policy has been updated), the AP Metrics response should pass the following additional verifications:
+For the second response (after the policy has been updated), the AP Metrics Response should pass the following additional verifications:
+
+* Verify that the agent includes one Associated STA Traffic Stats TLV for the associated STA.
+* Verify that the Associated STA Traffic Stats TLV is decodable and includes all fields per TLV definition in [1]
+* Verify that the agent includes one Associated STA Link Metrics TLV for the associated STA.
+* Verify that the Associated STA Link Metrics TLV is decodable and includes all fields per TLV definition in [1]
+
+
+## 4.7.6 MAUT Per-AP Metrics Response Controlled by AP Metrics Channel Utilization Reporting Threshold test
+
+This test verifies that a MAUT correctly sends AP and STA related metrics in the AP Metrics Response message based on the AP Metrics Channel
+Utilization Reporting Threshold setting specified by the Controller in the Multi-AP Policy Config Request message.
```

Comment:
(nitpick) CODING_STYLE.md specifies one line per sentence.

```suggestion
This test verifies that a MAUT correctly sends AP and STA related metrics in the AP Metrics Response message based on the AP Metrics Channel Utilization Reporting Threshold setting specified by the Controller in the Multi-AP Policy Config Request message.
```

More importantly, though, I think the following additional sentences could help a lot to clarify what the test really does.

```
After setting the policy, a STA is connected to the agent and starts sending traffic.
This causes the channel utilisation to increase and exceed the configured threshold.
At that point, the agent must send an AP Metrics Response message to the controller.
```

Reply:
Done